tmeister/static/src/common/use-localstorage-as-toggle.hook.js

```diff
@@ -0,0 +1,23 @@
+import { useState, useEffect } from 'react'
+
+export default function useLocalStorageAsToggle(toggle) {
+  const [ toggleActive, setToggleActive ] = useState(false)
+  useEffect(() => {
+    if(toggle) {
+      try {
+        const value = localStorage.getItem(`tm-toggle:${toggle}`)
+        if (value != undefined) {
+          if (value === 'true' || value === 'on') {
+            setToggleActive(true)
+          } else {
+            setToggleActive(false)
+          }
+        }
+      } catch (err) {
+        setToggleActive(false)
+      }
+    }
+  }, [toggle])
+
+  return toggleActive
+}
```

Comment:
I need a way to add features to toggle meister without the back-end being done. So I'm using this to fake feature toggles for our feature toggles system